go/RELEASING.md

```
@@ -13,6 +13,7 @@ $ (cd go && go mod verify)
 $ (cd go && go test -race ./...)
 $ (cd go && go vet ./...)
 $ (cd go && golangci-lint run ./...)
+$ (cd go && go run golang.org/x/vuln/cmd/govulncheck@v1.7.0 ./...)
```

markdownlint-cli2 (0.23.2)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)